string agvId 
string msgType
int64 timestamp
int32 result 




